packages/tests/utils/BasePage.ts

```diff
@@ -81,8 +81,8 @@ export class BasePage<T extends HTMLElement = HTMLElement> {
     return this;
   }
 
-  makeSnapshot(name?: string, opts?: Parameters<typeof cy.percySnapshot>[1]) {
-    cy.percySnapshot(name, opts);
+  makeSnapshot(config?: Parameters<typeof cy.matchImage>[0]) {
```

Comment:
nice, however I think we can use types used by plugin 
https://www.npmjs.com/package/@frsource/cypress-plugin-visual-regression-diff?activeTab=code
worth to try (not sure if those are acessible, should be because types are saved in global cypress)

![image](https://github.com/vuestorefront/storefront-ui/assets/2566152/2d2f1495-f30d-42be-860b-65d663c4a3d3)


```suggestion
  makeSnapshot(config?: Cypress.MatchImageOptions) {
```
On the other hand if type placement will changed in plugin we will lose it here. Maybe @FRSgit export type in plugin ?:D

Reply:
Agreed, this looks much more clean @Szymon-dziewonski